Homework1/Burrows–WheelerTransform/TestsForBWT.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace BurrowsWheelerTransform;
+
+/// <summary>
+/// Класс содержит методы, которые тестируют Direct and Reverse BWT.
+/// </summary>
+internal class TestsForBWT
+{
+    /// <summary>
+    /// Метод проверяет корректность выполнения всех тестов.
+    /// </summary>
+    /// <returns> true, если тесты прошли успешно. false иначе. </returns>
+    internal static bool AreTestsPassing()
+    {
+        return TestWithNonrepeatingSymbols() && TestWithRussianAndRepeatingSymbols() && TestWithOneSymbol();
+    }
```

Comment:
```suggestion
    internal static bool AreTestsPassing()
        =>TestWithNonrepeatingSymbols() && TestWithRussianAndRepeatingSymbols() && TestWithOneSymbol();
```
Думаю, что такие тесты уже не актуальны, но всё же :)